extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -145,7 +147,13 @@ export class ResourceTypePickerDialog extends DialogBase {
 		this._toDispose.push(listView.onDidClick((e) => {
 			this._resourceSearchBox.value = '';
 			this.filterResources();
-			listView.focus();
+			if (this.focusCards) {
```

Comment:
I'm not understanding why this is necessary - in my testing it always focuses the list view anyways. Why would we only ever want this to happen once?

Reply:
yeah, the focus should stay on list view when click happens.